doc/introduction/compiling_circuits.rst

```diff
@@ -254,4 +254,48 @@ observables and coefficients:
 >>> coeffs_groupings
 [[0.97, 4.21], [1.43]]
 
-This and more logic to manipulate Pauli observables is found in the :mod:`~.pennylane.grouping` module.
\ No newline at end of file
+This and more logic to manipulate Pauli observables is found in the :mod:`~.pennylane.grouping` module.
+
+Simplifying Operators
```

Comment:
I think this should be moved higher up.  Either at the top or right below "Compilation transforms".

Reply:
Moved at the top.